zulip_bots/zulip_bots/bots/joinme/doc.md

```diff
@@ -0,0 +1,20 @@
+# Joinme bot
+
+The Joinme bot posts a link to a Joinme call when being mentioned.
+
+## Setup
```

Comment:
This section will need to contain specific information on how to retrieve authentication data (e.g. login to join.me, click on button x to see your api key).